packages/fxa-shared/db/models/auth/recovery-key.ts

```diff
@@ -81,6 +97,13 @@ export class RecoveryKey extends BaseAuthModel {
     return RecoveryKey.fromDatabaseJson(rows[0]);
   }
 
+  static async findHintByUid(uid: string) {
```

Comment:
By adding this here, it kind of elminates the need for the storaged procedure.

Reply:
I change this at Wil's suggestion yesterday (I was initially calling a new stored procedure for the `findHintByUid`) - I'll make a similar change for the `updateStorageHint` hint method.